lib/SMS.rb

```diff
@@ -0,0 +1,30 @@
+require 'twilio-ruby'
+class SMS
+
+  TIME_FORMAT = "%H:%M"
+
+  def initialize(config, client: nil)
+    @client = client || Twilio::REST::Client.new(config[:account_sid], config[:auth_token])
+    @config = config
```

Comment:
config parameters could be refactored with ENV variables for it to be called easier and safer to store since the credentials on your machine.